paper.tex

```diff
@@ -173,16 +173,20 @@ \section{Background}
 \subsection{Virtualization and Scheduling in the GPU programming model}
 Contemporary GPUs are hierarchically organized, massively parallel processors designed to prioritize throughput over latency. (one sentence on thread hierarchy maybe). As comprehensive descriptions of the GPU programming model~\cite{} already exist, we focus on the aspects most relevant to this work: scheduling and synchronization.
 
+\raph{I think "NVIDIA Tesla: A Unified Graphics and Computing Architecture" Lindholm 2008 might be a good choice for the above cite.}
```

Comment:
I have historically used that Lindholm reference for the GPU _hardware_, but for the programming model I have used Nickolls, "Scalable parallel programming with CUDA" (ACM Queue, Mar/Apr 2008).